packages/esm-patient-banner-app/src/banner/patient-banner.scss

```diff
@@ -16,7 +16,7 @@
 .buttonCol {
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
```

Comment:
This will affect all the places, please test this change everywhere, or make the change local to the component it is being used.